test/__mocks__/uuid.js

```
@@ -0,0 +1,4 @@
+// CJS shim for uuid v14 (pure-ESM) so Jest can load @solana/web3.js.
+// Only v4 is needed by rpc-websockets (request ID generation).
+const { randomUUID } = require('crypto');
```

ESLint

[error] Require statement not part of import statement.

(`@typescript-eslint/no-var-requires`)

---

[error] 'require' is not defined.

(no-undef)